README.md

```diff
@@ -1,4 +1,18 @@
 # Getting Started
+Pull this repo
+Go to 
+https://github.com/ikottman/slime_ai
+
+If you have github installed in your python IDE then use the command
```

Comment:
github is the site, the thing you use locally is just `git`.

Reply:
how do I update the code based on this?